cmd/minikube/cmd/start_flags.go

```diff
@@ -56,6 +56,7 @@ const (
 	humanReadableDiskSize   = "disk-size"
 	nfsSharesRoot           = "nfs-shares-root"
 	nfsShare                = "nfs-share"
+	virtiofsShare           = "virtiofs-share"
```

Comment:
Do we want --virtiofs-share or --virtiofs-shares?

The first is consistent with --nfs-share but the second is more correct. We can add --virtiofs-shares and add --nfs-shares as alias to --nfs-share.